LeaderboardBackend/Services/Impl/AccountRecoveryService.cs

```diff
@@ -55,8 +59,14 @@ public async Task<CreateRecoveryResult> CreateRecoveryAndSendEmail(User user)
                 GenerateAccountRecoveryEmailBody(user, recovery)
             );
         }
-        catch
+        catch (Exception e)
         {
+            _logger.LogError(
+                "Recovery email failed to send with {type} for user {id}, {username}",
```

Comment:
Could you alter it slightly to make it clear that first thing that's being logged is the type of exception? Something like "Encountered error {type}" or "threw {type}" would work.

Reply:
Actually just {type} at the start should do.